modules/Error/Error.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
```

Comment:
Do all these files need to be client components?

The `window.location.reload` can technically be replaced just by using empty `href=""` on the refresh link, that should trigger a page reload.

Would be much better to have this server side rendered.

Reply:
All the error pages should be client components according to the docs